docs/testing/README.md

```diff
@@ -30,15 +30,15 @@ To set up testing in your project manually, you will need to follow the instruct
 
 To help you get started with testing, we recommend [reading this article](https://dev.to/open-wc/testing-workflow-for-web-components-g73) for a great step by step guide.
 
-## Karma
+## Web Test Runner
 
-For testing in the browser, we recommend [Karma](https://karma-runner.github.io/latest/index.html). With Karma, you can run unit tests for our javascript, as well as component tests on the rendered dom elements.
+For testing in the browser, we recommend [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/). With Web Test Runner, you can run unit tests for our javascript, as well as component tests on the rendered dom elements.
```

Comment:
..., you can run unit tests on JavaScript code, as well as component tests on the rendered DOM elements.

Reply:
done